examples/pixart/controlnet_pixart_alpha.py

```diff
@@ -0,0 +1,292 @@
+from typing import Any, Dict, Optional
+
+import torch
+from torch import nn
+
+from diffusers.models.attention import BasicTransformerBlock
+from diffusers.models import PixArtTransformer2DModel
+from diffusers.configuration_utils import ConfigMixin, register_to_config
+from diffusers.models.modeling_utils import ModelMixin
+from diffusers.models.modeling_outputs import Transformer2DModelOutput
+
+class PixArtControlNetAdapterBlock(nn.Module):
```

Comment:
I think we need to copy paste all the model code here into the pipeline so that the pipeline will be able to run, no?

Reply:
the pipeline code changes the sys path
`sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))`